docs/user-guide/menu-bar.md

```diff
@@ -142,47 +138,51 @@ When all the properties have been set, selecting the <img src="../img/button/sav
 
 Once a bookmark has been created, it is always possible to *Edit* it <img src="../img/button/edit-service-icon.jpg" class="ms-docbutton"/> or *Remove* it <img src="../img/button/delete-service-icon.jpg" class="ms-docbutton"/> from the list.
 
-## Burger Menu
+## Options Bar
 
-The *Burger Menu* is an important tools container that allows the user to perform different operations and take a look at several information:
+The *Options Bar* is an important tools bar, positioned on the right side of the map viewer, that allows the user to perform different operations and take a look at several information:
 
-<img src="../img/menu-bar/burger-menu.jpg" class="ms-docimage" style="max-width:300px;"/>
+<img src="../img/menu-bar/options-menu.jpg" class="ms-docimage" style="max-width:300px;"/>
 
 In particular, with these options it is possible to:
 
-* [Print](print.md) the map
+* Go back to the [Homepage](https://mapstore.geosolutionsgroup.com/mapstore/#/) by clicking the <img src="../img/button/homepage.jpg" class="ms-docbutton"/> button
+
+* **Login/Logout** by clicking the <img src="../img/button/logged.jpg" class="ms-docbutton"/> button (for more information see the [Managing Users and Groups](managing-users-and-groups.md) section) 
+
+* Open the **Version** panel to see the information about the deployed version of [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) by clicking the <img src="../img/button/version.jpg" class="ms-docbutton"/> button
 
-* **Export map** in `json` format
+* [Print](print.md) the map by clicking the <img src="../img/button/print2.jpg" class="ms-docbutton"/> button
 
-* [Import](import.md) files from your machine
+* **Export map** in `json` format by clicking the <img src="../img/button/export2.jpg" class="ms-docbutton"/> button
 
-* Open the [Catalog](catalog.md) in order to connect to a remote service and add layers to the map
+* [Import](import.md) files from your machine by clicking the <img src="../img/button/import2.jpg" class="ms-docbutton"/> button
```

Comment:
* [Import](import.md) files from your computer by clicking the <img src="../img/button/import2.jpg" class="ms-docbutton"/> button